index.html

```diff
@@ -3993,15 +3981,190 @@ <h2><span class="chunk">mASt</span> mastering metadata</h2>
+            SDR-to-HDR conversion).
+          </aside>
+
+
+          <aside class="example">
+            Example <span class="chunk">mASt</span> chunk mastering display colour primaries for SDR:
+            <table id="mASt-chunk-sdr-primaries-example" class="numbered simple">
+              <tr>
+                <th>Name</th>
+                <th>Actual values</th>
+                <th>Stored values</th>
+              </tr>
+
+              <tr>
+                <td rowspan="3">Colour primaries specified in [[ITU-R BT.709]]</td>
+                <td>(0.640, 0.330)</td>
+                <td>{ 32000, 16500 }</td>
+              </tr>
+
+              <tr>
+                <td>(0.300, 0.600)</td>
+                <td>{ 15000, 30000 }</td>
+              </tr>
+
+              <tr>
+                <td>(0.150, 0.060)</td>
+                <td>{ 7500, 3000 }</td>
+              </tr>
+
+            </table>
+          </aside>
+
+          <aside class="example">
+            Example <span class="chunk">mASt</span> chunk mastering display white point for SDR:
+            <table id="mASt-chunk-sdr-white-point-example" class="numbered simple">
+              <tr>
+                <th>Name</th>
+                <th>Actual values</th>
+                <th>Stored values</th>
+              </tr>
+
+              <tr>
+                <td>Illuminant D65 specified in [[SMPTE RP 177]]</td>
+                <td>(0.3127, 0.3290)</td>
+                <td>{ 15635, 16450 }</td>
+              </tr>
+            </table>
+          </aside>
+
+          <aside class="example">
+            Example <span class="chunk">mASt</span> chunk mastering display maximum 
+            luminance using traditional SDR shading techniques at 100 cd/m<sup>2</sup> 
+            (described in ITU-R BT.2035):
+            <table id="mASt-chunk-max-luminance-example" class="numbered simple">
+              <tr>
+                <th>Actual value</th>
+                <th>Stored value</th>
+              </tr>
+
+              <tr>
+                <td>100 cd/m<sup>2</sup></td>
+                <td>100</td>
+              </tr>
+            </table>
+          </aside>
+
+          <aside class="example">
+            Example <span class="chunk">mASt</span> chunk mastering display maximum 
+            luminance using "single-master" SDR shading techniques at 200 cd/m<sup>2</sup> 
+            (described in ITU-R BT.2408 Annex Y):
+            <table id="mASt-chunk-max-luminance-example" class="numbered simple">
+              <tr>
+                <th>Actual value</th>
+                <th>Stored value</th>
+              </tr>
+
+              <tr>
+                <td>200 cd/m<sup>2</sup></td>
+                <td>200</td>
+              </tr>
+            </table>
+          </aside>
+
+
+
+
+        </section>
+      </section>
+      
+
+
+      <section id="mLUM-chunk">
+        <h2><span class="chunk">mLum</span> mastering metadata</h2>
```

Comment:
I had [suggested]()  Maximum and Frame Average light levels as the section title, which I think is clearer. "mastering metadata" is vague and applies equally to mASt.

Reply:
How about "Metadata for Maximum Single-Pixel  and Frame-Average Luminance Levels". Since this is a still graphic, there is only a single frame.  Maybe the description can identify this

Reply:
Since APNG is now supported, it is no longer a single frame.

Reply:
I've added a description with an exception for multiple frames (image sequence).